tests/integration_tests/dashboards/api_tests.py

```diff
@@ -1346,7 +1346,9 @@ def test_export(self):
         # freeze time to ensure filename is deterministic
         with freeze_time("2020-01-01T00:00:00Z"):
             rv = self.get_assert_metric(uri, "export")
-            headers = generate_download_headers("json")["Content-Disposition"]
+            headers = generate_download_headers(
```

Comment:
Can you leave this test unmodified and make it run with the feature flag off?

```python
    @patch.dict(
        "superset.extensions.feature_flag_manager._feature_flags",
        {"VERSIONED_EXPORT": False},
        clear=True,
)
```

And maybe remove the feature flag patch from the `test_export_bundle` test below on line 1406.

Reply:
Went through and removed this feature flag patch when set to true for a couple of tests.